src/EnergyPlus/PoweredInductionUnits.cc

```diff
@@ -2147,6 +2153,16 @@ namespace PoweredInductionUnits {
         }
     }
 
+    void PowIndUnitData::CalcOutdoorAirVolumeFlowRate()
+    {
+        // calculates zone outdoor air volume flow rate using the supply air flow rate and OA fraction
+        if (this->AirLoopNum > 0) {
+            this->OutdoorAirFlowRate = (DataLoopNode::Node(this->PriAirInNode).MassFlowRate / DataEnvironment::StdRhoAir) * DataAirLoop::AirLoopFlow(this->AirLoopNum).OAFrac;
+        } else {
+            this->OutdoorAirFlowRate = 0.0;
+        }
+    }
+
```

Comment:
The new member function for the series and parallel power inductin air terminal units.